apps/studio/components/interfaces/Settings/Database/IPv4ConnectionNotices.tsx

```diff
@@ -37,3 +36,32 @@ export const IPv4DeprecationNotice = () => {
     </Alert_Shadcn_>
   )
 }
+
+export const IPv4AddonDirectConnectionNotice = () => {
+  return (
+    <Alert_Shadcn_ variant="default">
+      <AlertTitle_Shadcn_>
+        Direct database connections is recommended if you're connecting with session mode.
+      </AlertTitle_Shadcn_>
+      <AlertDescription_Shadcn_ className="space-y-3">
+        <p>
+          We strongly recommend you connect directly to your database instead of connection pooling
```

Comment:
If you are not connecting to the database from serverless environments, we strongly recommend you connect to your database directly instead of using connection pooling for lower latency.

Since your project has the IPv4 add on, the connection pooler isn't required to connect to the database from IPv4 only environments. 

(wanted to add a note about serverless environments)